sorbet/tapioca/require.rb

```diff
@@ -9,6 +9,8 @@
 require "rails/generators"
 require "rails/generators/app_base"
 require "tapioca/dsl/compilers"
+require "tapioca/dsl/compilers/active_record_relations"
+require "tapioca/dsl/helpers/active_record_column_type_helper"
```

Comment:
Adding these explicit requires allows us to remove the Tapioca shim.

If you add `ActiveRecord.eager_load!` to the bottom of this file, you could probably get rid of `activerecord.rbi`.

If you require `attr_json` here and run `tapioca dsl`, you can probably get rid of the `attr_json` shim.

Reply:
Thanks! I've had mixed success orchestrating all of these requires in the past with our internal projects, and it's been easier to just cut the shim. But I'll give it a try here!